src/register.rs

```diff
@@ -85,6 +85,14 @@ impl Drop for Probe {
     }
 }
 
+/// An id associated with the credentials of the current process. After [registering](crate::Submitter::register_personality)
+/// the id, it can be [passed](crate::squeue::Entry::personality) into submission queue entries
+/// to issue the request with the process' credentials.
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub struct Personality {
```

Comment:
I think it's good to use u16 directly, which seems to be a unnecessary newtype.

Reply:
You got it, done in 940e80a